build/source/engine/checkStruc.f90

```diff
@@ -152,10 +152,10 @@ subroutine checkPopulated(iStruct,metadata,err,message)
 
   ! loop through variables
   do iVar=1,size(metadata)
-
+  
```

Comment:
Minor cleanup, but could you remove empty line spaces?

Reply:
done